Makefile

```diff
@@ -19,7 +19,7 @@ $(BIN): $(BIN).c
 
 $(BIN)-native: $(BIN).c
 	$(VECHO) "  CC+LD\t\t$@\n"
-	$(Q)$(CC) $(CFLAGS) -o $@ $< \
+	$(Q)$(CC) $(CFLAGS) -m32 -o $@ $< \
```

Comment:
`amacc-native` is not intended to be fully functioned. Instead, it is just used for validating host environment and acting parsing purpose. Therefore, don't make changes against Makefile.